packages/cmrjs/utils.js

```diff
@@ -155,6 +155,15 @@ async function updateToken(cmrProvider, clientId, username, password) {
   // Update the saved ECHO token
   // for info on how to add collections to CMR: https://cmr.earthdata.nasa.gov/ingest/site/ingest_api_docs.html#validate-collection
   let response;
+  const ip = await publicIp.v4()
```

Comment:
I think it would be nice to move this to its own function and unit test that function